helm/kanister-operator/templates/deployment.yaml

```diff
@@ -53,6 +61,10 @@ spec:
       - name: {{ template "repository-server-controller.name" . }}
         image: {{ .Values.repositoryServerControllerImage.registry }}/{{ .Values.repositoryServerControllerImage.name }}:{{ .Values.repositoryServerControllerImage.tag }}
         imagePullPolicy: {{ .Values.repositoryServerControllerImage.pullPolicy }}
+{{- if .Values.controller.containerSecurityContext }}
+        securityContext: 
+{{ toYaml .Values.controller.containerSecurityContext | indent 10 }}
+{{- end }}
```

Comment:
Let's create a function is `_helpers.tpl` and use that at both the places for containers. We can do the same for deployment as well.

Reply:
@anishbista60 I see you have resolved this comment, but I don't see the changes for the deployment security context.

Reply:
In previous feedback you told me to create `_helpers.tpl` for `containersecuritycontext`  which i already did. Now, the question is should i configure for `podsecuritylevel` right ?